ThunderCloud/StandardGridItem.swift

```diff
@@ -11,20 +11,21 @@ import ThunderCollection
 
 /// `StandardGridItem` is a subclass of `GridItem` it represents a row with an item with a title, description and image. It is an adapter for the object in the CMS. All logic is done on it's super.
 open class StandardGridItem: GridItem {
-
-	open override var cellClass: UICollectionViewCell.Type? {
-		return StandardGridItemCell.self
-	}
-	
-	open override func configure(cell: UICollectionViewCell, at indexPath: IndexPath, in collectionViewController: CollectionViewController) {
-		
-		guard let standardCell = cell as? StandardGridItemCell else { return }
-		
-		standardCell.imageView?.isHidden = image == nil
-		standardCell.imageView?.image = image
-		standardCell.titleLabel?.isHidden = title == nil || title!.isEmpty
-		standardCell.titleLabel?.text = title
-		standardCell.subtitleLabel?.isHidden = description == nil || description!.isEmpty
-		standardCell.subtitleLabel?.text = description
-	}
+    
+    open override var cellClass: UICollectionViewCell.Type? {
+        return StandardGridItemCell.self
+    }
+    
+    open override func configure(cell: UICollectionViewCell, at indexPath: IndexPath, in collectionViewController: CollectionViewController) {
+        
+        guard let standardCell = cell as? StandardGridItemCell else { return }
+        
+        standardCell.imageView?.accessibilityLabel = image?.accessibilityLabel
+        standardCell.imageView?.isHidden = image == nil
+        standardCell.imageView?.image = image?.image
+        standardCell.titleLabel?.isHidden = title == nil || title!.isEmpty
```

Comment:
Could `title == nil || title!.isEmpty` alternatively be `title?.isEmpty ?? true`. Appreciate it's a really small change & that the force unwrapping should be fine here, but would be nice to remove it anyway where possible :D